File: app/src/main/java/App/repositories/retrofit/responses/UsuariosResponse.java
package App.repositories.retrofit.responses;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


import java.util.List;

import App.repositories.room.entities.Usuarios;

public class UsuariosResponse {



    @SerializedName("status")
    @Expose
    private String status;

    @SerializedName("message")
    @Expose
    private String message;

    @SerializedName("data")
    @Expose
    private List<Usuarios> usuarios = null;

    public UsuariosResponse(String status, String message, List<Usuarios> usuarios) {
        this.status = status;
        this.message = message;
        this.usuarios = usuarios;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<Usuarios> getUsuarios() {
        return usuarios;
    }

    public void setUsuarios(List<Usuarios> usuarios) {
        this.usuarios = usuarios;
    }
    
}

File: app/src/main/java/App/repositories/retrofit/responses/UsuariosResponse.kt
package App.repositories.retrofit.responses

import App.repositories.room.entities.Usuarios
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class UsuariosResponse(@field:Expose @field:SerializedName("status") var status: String, @field:Expose @field:SerializedName("message") var message: String, usuarios: List<Usuarios>?) {

    @SerializedName("data")
    @Expose
    var usuarios: List<Usuarios>? = null

    init {
        this.usuarios = usuarios
    }
}